### ISSUE TITLE
Is this safe?

### ISSUE THREAD

**Issue by BemusedHacker** [VenTaz/Vape#19]

**Comment by 7GrandDadPGN** [comment]
Yes, It is safe from the time it still worked, All it did was fake the authentication and stream the Vape DLL, The source to the server will be released soon, Please close this issue.

**Comment by BemusedHacker** [comment]
But why are the names quite weird? I mean its called "will u still think its infected?"

**Comment by 7GrandDadPGN** [comment]
Those are not the names, they are the comments, and its because people were calling it a RAT (which is common for cracks)
It is use at your own risk, Yes it does work but the person running the server CAN upload a malicious DLL (unlikely)
However now that the server is being released, you can run it on your own computer making it 100% secure and NOT a RAT.
